runelite-client/src/main/java/net/runelite/client/plugins/bankhistory/BankHistoryPanel.java

```diff
@@ -0,0 +1,622 @@
+
+		//Date picker
+		LocalDate today = LocalDate.now();
+		LocalDateTime startOfDay = today.atStartOfDay();
+		Consumer<Void> callback = (t) ->
+			updateDataset((String) accountSelectionCombo.getSelectedItem());
+
+		startDatePickerPanel = new DatePickerPanel(startOfDay, "Start Date", callback);
+		toDatePickerPanel = new DatePickerPanel(startOfDay.plusDays(1), "End Date", callback);
+		startDatePickerPanel.init();
+		toDatePickerPanel.init();
+
+		JPanel datePickerContainer = new JPanel();
+		datePickerContainer.setLayout(new BoxLayout(datePickerContainer, BoxLayout.PAGE_AXIS));
+		datePickerContainer.setBackground(ColorScheme.DARKER_GRAY_COLOR);
+		datePickerContainer.add(startDatePickerPanel);
+		datePickerContainer.add(Box.createRigidArea(new Dimension(0, 10)));
+		datePickerContainer.add(toDatePickerPanel);
+		datePickerContainer.setVisible(false);
+		showAdvancedButton.addMouseListener(new MouseAdapter()
+		{
+			@Override
+			public void mouseClicked(MouseEvent e)
+			{
+				boolean visible = datePickerContainer.isVisible();
+				datePickerContainer.setVisible(!visible);
+				simpleContainer.setVisible(visible);
+				if (visible)
+				{
+					showAdvancedButton.setText("Advanced");
+					timeSelection = null;
+				}
+				else
+				{
+					showAdvancedButton.setText("Simple");
+				}
+			}
+		});
+		//Advanced button
+		JPanel advancedContainer = new JPanel();
+		advancedContainer.setLayout(new BorderLayout());
+		advancedContainer.add(showAdvancedButton, BorderLayout.CENTER);
+		advancedContainer.add(Box.createRigidArea(new Dimension(0, 10)), BorderLayout.SOUTH);
+		advancedContainer.setMaximumSize(new Dimension(1280, 45));
+
+		//Open in new window
+		JPanel openInNewWindowContainer = new JPanel();
+		openInNewWindowContainer.setLayout(new BorderLayout());
+		openInNewWindowContainer.add(Box.createRigidArea(new Dimension(0, 10)), BorderLayout.SOUTH);
+		JButton newWindowButton = new JButton("Open In New Window");
+		newWindowButton.setMaximumSize(new Dimension(100, 30));
+		newWindowButton.setFocusPainted(false);
+		newWindowButton.addMouseListener(new MouseAdapter()
+		{
+			@Override
+			public void mouseReleased(MouseEvent e)
+			{
+				JFrame frame = (JFrame) SwingUtilities.getWindowAncestor(BankHistoryPanel.this);
+				JDialog dialog = new JDialog(frame, "Bank History", false);
+				BankHistoryPanel panel = new BankHistoryPanel();
+				panel.setTracker(tracker);
+				panel.setPlugin(plugin);
+				panel.init(true);
+				dialog.setPreferredSize(new Dimension(500, 500));
+
+				dialog.getContentPane().add(panel, BorderLayout.CENTER);
+
+				dialog.setLocationRelativeTo(null);
+				dialog.pack();
+				dialog.setVisible(true);
+			}
+		});
+
+		openInNewWindowContainer.add(newWindowButton, BorderLayout.CENTER);
+
+		//bank tab selection
+		JPanel tabPanel = new JPanel();
+		JComboBox<String> tabSelectionCombo = new JComboBox<>();
+		tabSelectionCombo.setModel(getComboBoxModel(DatePickerPanel.getArrayOfIntegers(0, 8, false)));
```

Comment:
```suggestion
		tabSelectionCombo.setModel(getComboBoxModel(DatePickerPanel.getArrayOfIntegers(0, 10, false)));
```

Should be 0, 10 instead otherwise all bank tabs will not show up in the bank tab dropdown.